tests/methods/test_lime_text.py

```diff
@@ -165,21 +102,24 @@ def tokenizer():
     ('UNKWORDZ a bad UNKWORDZ UNKWORDZ!?\'"', 9),
     ('such UNKWORDZ UNKWORDZ movie "UNKWORDZUNKWORDZ\'UNKWORDZ', 9),
     ('such a bad UNKWORDZ UNKWORDZ!UNKWORDZ\'UNKWORDZ', 9),
-    pytest.param('its own self-UNKWORDZ universe.', 7,
+    pytest.param('its own self-UNKWORDZ universe.',
+                 7,
                  marks=pytest.mark.xfail(reason='poor handling of -')),
-    pytest.param('its own UNKWORDZ-contained universe.', 7,
+    pytest.param('its own UNKWORDZ-contained universe.',
+                 7,
                  marks=pytest.mark.xfail(reason='poor handling of -')),
-    pytest.param('Backslashes are UNKWORDZ/cool.', 6,
+    pytest.param('Backslashes are UNKWORDZ/cool.',
+                 6,
                  marks=pytest.mark.xfail(reason='/ poor handling of /')),
-    pytest.param('Backslashes are fun/UNKWORDZ.', 6,
+    pytest.param('Backslashes are fun/UNKWORDZ.',
+                 6,
                  marks=pytest.mark.xfail(reason='poor handling of /')),
-    pytest.param('    ', 0,
-                 marks=pytest.mark.xfail(reason='Repeated whitespaces')),
-    pytest.param('I like   whitespaces.', 4,
+    pytest.param(
+        '    ', 0, marks=pytest.mark.xfail(reason='Repeated whitespaces')),
+    pytest.param('I like   whitespaces.',
+                 4,
                  marks=pytest.mark.xfail(reason='Repeated whitespaces')),
 ])
```

Comment:
I don't really see how this is an improvement, but I'm fine with it either way :-)

Reply:
Those are done by the `pre-commit` command, not by me intentionally 😂 .